app/assets/javascripts/libs/backbone_wrapper.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import React from "react";
+
+type Props = {
+  backboneView: any,
+};
+
+class BackboneWrapper extends React.Component<Props> {
+  container: ?HTMLDivElement;
+
+  componentDidMount() {
+    if (this.container) this.container.appendChild(this.props.backboneView.render().el);
+  }
```

Comment:
`componentWillUnmount` to clean up event handlers?
Pass in props on `componentDidReceiveProps`?